### ISSUE TITLE
Generate wip changelogs

### ISSUE THREAD

**Issue by Isigiel** [conventional-changelog/standard-version#111]
I'd love to have a way to generate a master/upcoming chnages changelog, so even if there was no release the changelog would show the data from issues snce the latest release

**Comment by Tapppi** [comment]
This is a great idea! A related conversation, which I unfortunately noticed long after the actual conversation, was had in #83, a PR for allowing pre-releases, some time ago.

Anyway, if I understood correctly, this could be a sort of precursor to that. Do you want/have time to take a shot at a PR?

**Comment by Isigiel** [comment]
I hacked up a version for me https://gist.github.com/Isigiel/4c621cff3f58fd1983a6877b51bf0359
the major downside is that the commit fails if it's run without any new changelog items. also this version uses os specific line endings

**Comment by Tapppi** [comment]
@Isigiel Just to clarify, you have also modified `conventional-changelog` to return the title as `Unreleased vX.Y.Z`. PR's would be greatly appreciated, otherwise we'll get to it once we can! 😅

**Comment by jbottigliero** [comment]
Hi – I think the `--dry-run` option would (now) be an option for you – see example output below.

```bash
$ npx standard-version --dry-run
npx: installed 207 in 7.154s
✔ bumping version in package.json from 7.1.0 to 8.0.0
✔ outputting changes to CHANGELOG.md

---
## [8.0.0](https://github.com/conventional-changelog/standard-version/compare/v7.1.0...v8.0.0) (2020-01-25)


### ⚠ BREAKING CHANGES

* `composer.json` and `composer.lock` will no longer be read from or bumped by default. If you need to obtain a version or write a version to these files, please use `bumpFiles` and/or `packageFiles` options accordingly.

### Bug Fixes

* composer.json and composer.lock have been removed from default package and bump files. ([c934f3a](https://github.com/conventional-changelog/standard-version/commit/c934f3a38da4e7234d9dba3b2405f3b7e4dc5aa8)), closes [#495](https://github.com/conventional-changelog/standard-version/issues/495) [#394](https://github.com/conventional-changelog/standard-version/issues/394)
* **deps:** update dependency conventional-changelog to v3.1.18 ([#510](https://github.com/conventional-changelog/standard-version/issues/510)) ([e6aeb77](https://github.com/conventional-changelog/standard-version/commit/e6aeb779fe53ffed2a252e6cfd69cfcb786b9ef9))
* **deps:** update dependency yargs to v15.1.0 ([#518](https://github.com/conventional-changelog/standard-version/issues/518)) ([8f36f9e](https://github.com/conventional-changelog/standard-version/commit/8f36f9e073119fcbf5ad843237fb06a4ca42a0f9))
---

✔ committing package.json and CHANGELOG.md
✔ tagging release v8.0.0
ℹ Run `git push --follow-tags origin master && npm publish` to publish
```

I'm going to go ahead and close this issue since it's been open for some time with no activity and I think we have it solved – thanks!✌️
